src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -71,25 +67,28 @@ class CompetitionRobotContainer {
 
     AutoBuilder.configureHolonomic(
         m_chassis::getFusedPose, // Robot pose supplier
-        m_chassis
-            ::resetPose, // Method to reset odometry (will be called if your auto has a starting
+        m_chassis::resetPose, // Method to reset odometry (will be called if your auto has a
+        // starting
         // pose)
```

Comment:
Ugh. Not something you need to change, but I don't like how spotless reflows comments into 2 lines, but doesn't merge with the line underneath. I guess that's what `/* * /` block comments are for. oh well. It'll be ugly for now